src/course/syllabus/server/schedule.md

```diff
@@ -22,31 +22,26 @@ schedule:
       url: /workshops/node-express-server/
       type: workshop
   tuesday:
-    - name: Anni's talk
-      start: 11:00
-      end: 12:30
     - name: Server-side forms
-      start: 12:30
+      start: 11:00
       end: 13:00
       url: /workshops/server-side-forms/
       type: workshop
-    - name: Server-side forms
+    - name: Tech for Better - Introduction
```

Comment:
The design for the schedule means long names don't fit particularly well. Could we abbreviate these to `TFB -`?